translations/de/common.json

```diff
@@ -1 +1,81 @@
-{}
+{
+  "subtitle": "Scannen und Überprüfen von digitalen COVID-Zertifikaten der EU",
+  "abort": "Abbrechen",
+  "save": "Sichern",
+  "close": "Schließen",
+  "continue": "Weiter",
+  "back": "Zurück",
+  "version": "Version",
+  "sponsor": "Unterstützen",
+  "privacy": "Datenschutzerklärung",
+
+  "imprint": "Impressum",
+  "imprint.information": "Informationen nach Sektion 5 TMG",
+  "imprint.legal": "RECHTLICHE HINWEISE",
+  "imprint.contact": "Kontaktinformationen",
+  "imprint.email": "E-Mail",
+
+  "vaccination": "Impfung",
+  "recovery": "Genesung",
+  "test": "Test",
+
+  "meta.description": "Überprüfen Sie digitale COVID-Zertifikate der EU mit Leichtigkeit und validieren Sie sie anhand der länderspezifischen und lokalen Vorschriften",
+
+  "information.purpose": "Was ist der Zweck von Covid Validator?",
+  "information.purpose.message": "Mit Covid Validator können Sie digitale COVID-Zertifikate der EU auf der Grundlage nationaler oder lokaler Regeln scannen und überprüfen. CovidValidator zeigt der prüfenden Person nur so wenige persönliche Informationen wie möglich an, um die Privatsphäre der Nutzenden zu schützen. Diese Web-App ist eine Alternative zu anderen bestehenden Produkten, da sie nicht nur in bestimmte App-Stores oder Ländern verfügbar ist.",
```

Comment:
`CovidValidator` -> `Covid Validator` um es einheitlich zu halten

Generell hätte ich auch nichts überall `CovidValidator zu schreiben`

Reply:
Ich mache jetzt mal Covid Validator draus, wir können uns ja noch überlegen/du kannst dir ja noch überlegen welche Schreibweise du mehr magst.

Reply:
Geländert in [`0a59c8f`](https://github.com/timokoenig/covid-validator/pull/16/commits/0a59c8f0c5857c9b062b6da61d2ceea8d65d886f).